app/src/main/java/ke/ac/ku/ledgerly/presentation/bill_reminders/BillReminderListScreen.kt

```
@@ -185,120 +242,54 @@ fun BillReminderListScreen(
-                style = Typography.labelSmall,
-                color = MaterialTheme.colorScheme.onSurfaceVariant,
-                fontSize = MaterialTheme.typography.labelSmall.fontSize * 0.8f
-            )
+            Column(horizontalAlignment = Alignment.End) {
+                Text(
+                    "Total Amount",
+                    style = MaterialTheme.typography.labelSmall,
+                    color = MaterialTheme.colorScheme.onSurfaceVariant
+                )
+                Text(
+                    "KES ${String.format("%.0f", summary.totalAmount)}",
```

detekt (1.23.8)

[warning] String.format("%.0f", summary.totalAmount) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)